src/disco/store/fd_pending_slots.c

```diff
@@ -1,8 +1,7 @@
 #include "fd_pending_slots.h"
 
 void *
-fd_pending_slots_new( void * mem, ulong lo_wmark ) {
-
+fd_pending_slots_new( void * mem ) {
```

Comment:
`seed` should be passed in to initialize the rng

Reply:
right, that's better, `seed` has been added